package.json

```diff
@@ -1,39 +1,39 @@
 {
-  "name": "@openx/cypress-parallel",
-  "version": "0.2.3",
-  "description": "Reduce up to 40% your Cypress suite execution time parallelizing the test run on the same machine.",
-  "main": "cli.js",
+  "name": "cypress-parallel-dev",
```

Comment:
why do we need to change name of package?

Reply:
apologies if [my question](https://github.com/openx/cypress-parallel/pull/2#issuecomment-2207325613) was not clear.  we could not find docs on how to contribute/test changes here and i was asking how to publish this so that testing can take place.  i presume `cypress-parallel-dev` was chosen to avoid naming conflicts with `cypress-parallel`.

Reply:
if we want to test new package  we can locally connect our repo with tests to locally update cypress-parallel plugin, to have brief view if everything is ok.
After if everything is ok, new version of plugin can be published (https://docs.npmjs.com/creating-and-publishing-private-packages), but it would be good to have 
```
"@openx/cypress-parallel": "0.2.3"
```
instead of `"@openx/cypress-parallel": "^0.2.3"`, so it won't be updated by somebody unintentionally 

Switch test branch in ui-unity to new version of plugin and run the whole regression.